data_model.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: datamodel
+permalink: /datamodel/
+---
+
+Data Model -- Draft
```

Comment:
Let's remove `Draft`. We could replace with `Proposal` but I think better not to.